packages/jupyter-ai/src/handler.ts

```diff
@@ -204,7 +204,13 @@ export namespace AiService {
     label: string;
   };
 
-  export type Field = TextField | MultilineTextField;
+  export type IntegerField = {
+    type: 'integer';
+    key: string;
+    label: string;
```

Comment:
As I discussed above, we might need to add an optional default value for fields.

Reply:
I've created a new issue to track this: https://github.com/jupyterlab/jupyter-ai/issues/325